.github/actions/build-step/action.yml

```diff
@@ -36,50 +36,45 @@ runs:
   using: "composite"
   steps:
     - name: Set up environment
+      working-directory: ${{ inputs.path }}
       id: setup
       shell: bash
       run: |
+        cp overlay-memfault.conf overlay-memfault-att.conf
```

Comment:
@jorgenmk Could you move adding **overlay-upload-modem-trace-to-memfault.conf** to the debug build while you are at it?

Reply:
Will add in separate PR